lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -409,12 +409,12 @@ def __init__(
         device=None,
         batch_size: int = 1,
         batch_at_a_time: bool = False,
-        local: bool = False,
+        use_partitioning_service: bool = True,
         aryn_api_key: str = "",
         aryn_partitioner_address: str = DEFAULT_ARYN_PARTITIONER_ADDRESS,
         use_cache=False,
     ):
-        if local:
+        if not use_partitioning_service:
```

Comment:
Could you switch around the logic for this? Would like to avoid the use of the negative `if not` if possible

Reply:
Is there a need for the if statement at all? You could set `device = choose_device(device)` regardless since it won't matter if the remote partitioner is being used?

Reply:
Changed logic around.

Reply:
Since we [haven't yet changed](https://github.com/aryn-ai/sycamore/blob/3d83de4f90dff2e2271b7971bc4f177538acc15b/lib/sycamore/sycamore/transforms/detr_partitioner.py#L81) the `ArynPDFPartitioner` to conditionally load the model, this makes sure we don't take up GPU memory. I'll make another pr to fix this.